hw/ip/kmac/doc/_index.md

```diff
@@ -18,8 +18,8 @@ See that document for integration overview within the broader OpenTitan top leve
 - 64b x 9 depth Message FIFO
 - 1600b of internal state (internally represented in two shares for 1st-order masking)
 - Performance goals of >= 72 Mb/s @ 100MHz (when entropy is available always)
-    - SHA3-512: roughly 88 MB/s at most
-    - SHA3-224: 160 MB/s at most
+    - SHA3-512: roughly 66 MB/s at most
+    - SHA3-224: 120 MB/s at most
```

Comment:
@eunchan , I wasn't sure how the throughput for SHA3 should be computed based on the number of rounds and cycles per rounds. What I did now is just scale down by 33% as a single round now takes 4 instead of 3 clock cycles, but I don't know if that's accurate. Do you have a formula for this somewhere?